packages/vite-plugin-cloudflare/src/cloudflare-environment.ts

```diff
@@ -178,7 +156,29 @@ export function createCloudflareEnvironmentOptions(
 					external: [...cloudflareBuiltInModules],
 				},
 			},
-			webCompatible: true,
```

Comment:
They've also removed `webCompatible` and kind of replaced it with `keepProcessEnv` (https://github.com/vitejs/vite/pull/18395#discussion_r1820503185)